ddtrace/contrib/aioredis/patch.py

```diff
@@ -157,7 +157,7 @@ async def traced_13_execute_pipeline(func, instance, args, kwargs):
 
     cmds = []
     for _, cmd, cmd_args, _ in instance._pipeline:
-        parts = [cmd.decode() if isinstance(cmd, bytes) else cmd]
+        parts = [cmd.decode(errors="backslashreplace") if isinstance(cmd, bytes) else cmd]
```

Comment:
Was it also necessary to change this decode for linked issue or is this here to be consistent across the patching code in this integration?

Reply:
It's there to be consistent. The exceptions I was seeing came from the `traced_13_execute_command` function.

Looking at the original source more closely, this is probably not necessary and I can remove it.

Reply:
Removed this since it's unnecessary as far as I can tell.